src/Rector/Closure/ServiceArgsToServiceNamedArgRector.php

```diff
@@ -156,11 +156,13 @@ private function resolveConstructorParameterNames(string $serviceClass): array
 
         $extendedMethodReflection = $serviceClassReflection->getConstructor();
 
-        $parametersAcceptor = ParametersAcceptorSelector::selectSingle($extendedMethodReflection->getVariants());
+        $parametersAcceptorWithPhpDocs = ParametersAcceptorSelector::selectSingle(
+            $extendedMethodReflection->getVariants()
+        );
```

Comment:
You can use:

```php
ParametersAcceptorSelector::combineAcceptors(
$extendedMethodReflection->getVariants())
```

That ensure multi variants is supported.

Reply:
Done